starter-code/index.html

```diff
@@ -5,40 +5,76 @@
   </head>
   <body>
-    Premium Discover Help Download
-
-    Music for everyone.
-    Spotify is now free on mobile, tablet and computer.
-    Listen to the right music, wherever you are.
-
-    What’s on Spotify?
-
-    Millions of Songs
-    There are millions of
-    songs on Spotify
-
-    HD Music
-    Listen to music as if you
-    were listening live
-
-    Stream Everywhere
-    Stream music on your
-    smartphone, tablet or
-    computer
-
-    It’s as yeezy as Kanye West.
-    Search
-    Know what you want to listen to?
-    Just search and hit play.
-    Browse
-    Check out the latest charts,
-    brand new releases and great
-    playlists for right now.
-    Discover
-    Enjoy new music every Monday
-    with your own personal playlist.
-    Or sit back and enjoy Radio.
+    <nav class="nav-bar">
+      <div><img src="images/spotify-logo.png" alt="spotify-logo"></div>
+      <ul>
+        <li>Premium</li>
+        <li>Discover</li>
+        <li>Help</li>
+        <li>Download</li>
+      </ul>
+    </nav>
+    <header class="header">
+    <div class="conteiner">
+      <h1>Music for everyone.</h1>
+      <h2>Spotify is now free on mobile, tablet and computer.</h2>
+      <h2>Listen to the right music, wherever you are.</h2>
+    </div>
+    </header>
+    <!--section 1-->
+    <section id="section1">
+   
+      <h3>What’s on Spotify?</h3>
+      <div class="icons">
+        <article class="service-box">
+          <img src="images/music-icon.png" alt="music-icon">
+          <h4>Millions of Songs</h4>
+          <p>There are millions of</p>
+          <p>songs on Spotify</p>
+        </article>
+        <article class="service-box">
+          <img src="images/high-quality-icon.png" alt="quality-icon">
+          <h4>HD Music</h4>
+          <p>Listen to music as if you</p>
+          <p>were listening live</p>
+        </article>
+        <article class="service-box">
+          <img src="images/devices-icon.png" alt="devices-icon">
+          <h4>Stream Everywhere</h4>
+          <p>Stream music on your</p>
+          <p>smartphone, tablet</p><p>or computer</p>
+        </article>
+      </div>
+    </section>
+    <!--section 2-->
+    <section class="green-box">
+      <div class="text-box">
+        <h3>It’s as yeezy as Kanye West.</h3>
+        <article>
+          <h4>Search</h4>
+          <p>Know what you want to listen to?</p>
+          <p>Just search and hit play.</p>
+        </article>
+        <article>
+          <h4>Browse</h4>
+          <p>Check out the latest charts,</p>
+          <p>brand new releases and great</p>
+          <p>playlists for right now.</p>
+        </article>
+        <article>
+          <h4>Discover</h4>
+          <p>Enjoy new music every Monday</p>
+          <p>with your own personal playlist.</p>
+          <p>Or sit back and enjoy Radio.</p>
+        </article>
+      </div>
+      <div class="img-box">
+        <img src="images/spotify-app.jpg" alt="app-img">
+      </div>
```

Comment:
Here the structure should change a little bit.

I put a better structure below, the reason for this, is that you can see that the box and the image on the left appear below the "yeezy" title.
It's a little detail but it changes the structure a little bit.

And this will of course change the css a little bit

 <section class="green-box">
      <h3>It’s as yeezy as Kanye West.</h3>
      <div class="wrapper">
        <div class="text-box">
          <article>
            <h4>Search</h4>
            <p>Know what you want to listen to?</p>
            <p>Just search and hit play.</p>
          </article>
          <article>
            <h4>Browse</h4>
            <p>Check out the latest charts,</p>
            <p>brand new releases and great</p>
            <p>playlists for right now.</p>
          </article>
          <article>
            <h4>Discover</h4>
            <p>Enjoy new music every Monday</p>
            <p>with your own personal playlist.</p>
            <p>Or sit back and enjoy Radio.</p>
          </article>
        </div>
        <div class="img-box">
          <img src="images/spotify-app.jpg" alt="app-img" />
        </div>
      </div>
    </section>